### PR TITLE
Fixed counters bugs

### PR DESCRIPTION
**Pull request by trurl123**
Could not increment by 2
Could not pass variables to increment counters

### CODE REVIEW

File: src/Cassandra/Data/Linq/CqlExpressionVisitor.cs [hunk 1]
```diff
@@ -909,6 +924,19 @@ protected override Expression VisitMember(MemberExpression node)
...
+                        if (column == null)
+                            break;
+                        if (column.IsCounter)
+                        {
+                            var value = Expression.Lambda(node).Compile().DynamicInvoke();
+                            if (!(value is long || value is int))
+                            {
+                                throw new ArgumentException("Only Int64 and Int32 values are supported as counter increment of decrement values");
+                            }
+							_projections.Add(Tuple.Create(column, value, ExpressionType.Increment));
```
Comment (jorgebay) [inline src/Cassandra/Data/Linq/CqlExpressionVisitor.cs]: Use space indentation here :)

**Comment by jorgebay** [comment]
Apart from the minor nit about indentation, we should add an integration test for it, but otherwise it looks good to me.

**Comment by jorgebay** [comment]
Ticket: [CSHARP-281](https://datastax-oss.atlassian.net/browse/CSHARP-281)

**Comment by trurl123** [comment]
fixed identation

**Comment by landim** [comment]
Created another branch with the fix (provided by @trurl123) and with integration tests.
https://github.com/datastax/csharp-driver/pull/239

**Comment by jorgebay** [comment]
Merged #239 that includes an integration test.

Thanks @trurl123!

**Pull request closed without merging**